sky/adaptors/ibm.py

```diff
@@ -48,6 +55,15 @@ def get_api_key():
     return read_credential_file()['iam_api_key']
 
 
+def get_hmac_keys():
+    cred_file = read_credential_file()
```

Comment:
you're right with respect to duplicate code, but we'll have to use `adaptors/ibm.py`'s `read_credential_file()` to avoid circular import. I'll use the version found in `clouds/ibm.py`.

Reply:
Sounds great!